src/main/java/core/basesyntax/service/OperationService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationService {

```

Comment:
OperationHandler name can be better for this interface and you can move it to package core.basesyntax.strategy.impl